Apps/WebClient/src/Server/Controllers/RobotsController.cs

```diff
@@ -58,5 +60,20 @@ public ActionResult Robots()
             };
             return result;
         }
+
+        [SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "Team decision")]
+        private static string? ReadRobotsFile(string robotsFilePath)
```

Comment:
Nit: this could be made async similar to `ConfigurationService.ReadFeatureToggleConfigurationAsync()` and called within the `Robots()` action.

Reply:
Changed.